### ISSUE TITLE
The last element in a SelectBox cannot be reached in the Inspector in certain constellations

### ISSUE THREAD

**Issue by KPeschke** [neos/neos-ui#2683]
### Description
The last element in a SelectBox disappears behind the inspector action buttons and thus becomes inaccessible.

### Steps to Reproduce
1. Inspector-TabNavigation needs to be 2 lines
2. DropDown is last element
3. Browser window height is smaller than the Inspector tab panel
4. Open dropdown and scroll down to select the last element

#### Expected behavior
The last selection option can be reached without problems.
![image](https://user-images.githubusercontent.com/8969952/76232297-d0d32800-6226-11ea-8b98-2f771324cfd5.png)

#### Actual behavior
- Browser height ok, all is visible and reachable
![image](https://user-images.githubusercontent.com/8969952/76232150-95385e00-6226-11ea-828a-bb934be29254.png)

- Browser height to small
![image](https://user-images.githubusercontent.com/8969952/76232244-bc8f2b00-6226-11ea-8373-f75b4dcd489e.png)

### Affected Versions
Neos: 4.3.11 and previous
UI: 3.8.1 and previous

### Simple bugfix could be:
`.style__tabs__content___OZ4nq {
    // height: calc(100% - 141px);
    height: calc(100% - 184px);
    ...
}`

or 
`.style__inspectorTabPanel___2z7Yl {
    ...
    // height: 100%;
    height: calc(100% - 43px);
    ...
}`

**Comment by grebaldi** [comment]
Hi @KPeschke ,

this may have been fixed by https://github.com/neos/neos-ui/pull/2854. Is this issue still reproducable for you in a current Neos >= 5.3?

**Comment by Rasmizzle** [comment]
Hi,
in my opinion this is not a problem specific to select-boxes, but to longer lists of properties in the inspector in general as soon as there are more than six inspector tabs and it breaks to a new line. At least in Neos >= 7.2 I cannot recreate the issue above concerning hidden select box options but cut off inspector tab content is still a thing. Not sure if this is worth its own issue so I'll post a description down below:

### Description

When there are more than six tabs in the inspector the height of inspector-tab-content does not get calculated correctly. This can lead to properties not shown (fully) in the selected inspector tab.

### Steps to Reproduce

1. Inspector tab navigation has more than two lines
2. browser window height is smaller than the inspector tab panel content

#### Expected behavior

The last property in the inspector tab content is fully visible.

#### Actual behavior

When browser window is smaller than the height of tab content the content might get cut off. See screens attached. 
![Bildschirm­foto 2023-01-16 um 19 50 43](https://user-images.githubusercontent.com/47476377/212748301-fcc15aaf-e3d5-44d8-acb4-da8299363f05.png)

![Bildschirm­foto 2023-01-16 um 19 52 03](https://user-images.githubusercontent.com/47476377/212748430-75b9c6d1-5d04-409a-a736-fea3a92e8574.png)

The issue appears to be caused by the height of `.style__tabs__content___OZ4nq` being hardcoded with `height: calc(100% - 141px);` in compiled version of neos-ui.

### Affected Versions

Neos: 7.2.9

UI: 7.2.8

**Comment by grebaldi** [comment]
Thanks for the update, @Rasmizzle!

I can reproduce now and will look into the issue :)
